src/chrome/content/rules/Bullguard.com.xml

```diff
@@ -1,6 +1,24 @@
+<!--
+	Unable to connect:
+		media.bullguard.com
+	Invalid Certificate:
+		safebrowsing.bullguard.com
+		wwworigin.bullguard.com
+	Unable to get Local Issuer certificate:
```

Comment:
Please use something like `incomplete certificate chain` for this kind of issues.